### PR TITLE
CREATE: created the user entities file, added relevant variables and …

### PR DESCRIPTION
**Pull request by Dhyey618**
CREATE: entities file for the user information, including getters and setters.

### CODE REVIEW

File: backend/src/main/java/com/skysocial/backend/entities/Users.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    @Size(min = 2 , max =50)
+    private String firstName;
+    @Size(min = 2 , max =50)
+    private String lastName;
+    @Size(min = 2 , max =50)
+    private String userName;
```
Comment (akhil-maram) [inline backend/src/main/java/com/skysocial/backend/entities/Users.java]: Does our MVP include separate usernames or just first names / emails as usernames?

File: backend/src/main/java/com/skysocial/backend/entities/Users.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        this.password = password;
+        this.email = email;
+        this.phone = phone;
+        this.profileImage = profileImage;
+
+    }
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```
Comment (akhil-maram) [inline backend/src/main/java/com/skysocial/backend/entities/Users.java:23]: The @Data annotation should take care of the getter and setter for id

File: backend/src/main/java/com/skysocial/backend/entities/Users.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+import jakarta.persistence.Id;
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+
+
+@Entity
+@Data
+public class Users implements java.io.Serializable {
```
Comment (akhil-maram) [inline backend/src/main/java/com/skysocial/backend/entities/Users.java:19]: Look into the annotations for constructors to use (i.e. NoArgsConstructor)

**Review by akhil-maram — APPROVED** [review]

File: backend/src/main/java/com/skysocial/backend/entities/Users.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Users implements java.io.Serializable {
+    public Users() {
```
Comment (OliverSinnassamy) [inline backend/src/main/java/com/skysocial/backend/entities/Users.java]: Don't need the constructors when using annotations

**Review by OliverSinnassamy — APPROVED** [review]

**Pull request merged**